data/strings/strings.txt

```diff
@@ -11632,7 +11639,7 @@
     fa = اضافه کردن یک تجارت
     fi = Lisää organisaatio
     fr = Ajouter une organisation
-    hu = Szervezet hozzáadása
+    hu = Üzlet hozzáadása
```

Comment:
Nem biztos, hogy üzletet adsz hozzá

Reply:
Ez így oké, de ez oda-vissza is feltehető :)